pyproject.toml

```
@@ -10,20 +10,21 @@ authors = [
     { name = "Shuntaro C. Aoki", email = "s_aoki@i.kyoto-u.ac.jp" }
 ]
 readme = "README.md"
-requires-python = ">= 3.6, < 3.12"
+requires-python = ">= 3.6, < 3.9"
```

3.6 through 3.11 should be supported, so please don't remove support for higher versions

```suggestion
requires-python = ">= 3.6, < 3.12"
```